organization/mission.md

```diff
@@ -3,10 +3,12 @@
 ## Our mission
 
 ```{epigraph}
-2i2c’s mission is to make research and education more **impactful**, **accessible**, and **delightful** by developing, operating, and supporting open infrastructure for interactive computing.
+2i2c enables communities to co-create discoveries and learning through shared data, computing, and tools.
```

Comment:
I agree we need to update this mission statement and incorporate the multiple rounds of discussions and feedback we done into a new statement. 

In the strategy document, we had come up with 

_2i2c’s mission is to **grow the world’s collective ability to make discoveries and learn from one another** by developing, operating, and supporting open infrastructure for interactive computing._

I thought that was good because it described a long-term goal that emphasized 2i2c potential impact as a mission driven organization.

This revised mission statement doesn't carry that same sense of ambition. It feels more of a description of what offer today and not a future state we should be striving towards.